src/zookeeper/deploy/service.yaml

```diff
@@ -20,6 +20,7 @@ cluster-type:
 
 prerequisite:
   - cluster-configuration
+  - docker-cache
```

Comment:
You can ignore services with `cluster-type: yarn` because they are not supported now.